bazel/bazel.go

```diff
@@ -69,17 +71,38 @@ func (b *bazel) WriteToStdout(v bool) {
 	b.writeToStdout = v
 }
 
-func (b *bazel) newCommand(command string, args ...string) {
+func (b *bazel) newCommand(command string, args ...string) (*bytes.Buffer, *bytes.Buffer) {
 	b.ctx, b.cancel = context.WithCancel(context.Background())
 
 	args = append([]string{command}, args...)
+
+	if b.writeToStderr || b.writeToStdout {
+		containsColor := false
```

Comment:
how is this color change related to the rest of the PR? could it be separated out?

Reply:
It isn't. I'm pretty sure this is a leftover remnant from an internal modification to make sure ibazel always used color when running bazel commands. It can be pulled out.